billy/web/api/handlers.py

```diff
@@ -469,9 +481,27 @@ def read(self, request, abbr, chamber=None):
 class BoundaryHandler(BillyHandler):
     base_url = settings.BOUNDARY_SERVICE_URL
 
+    def _ocd_id_to_shape_url(self, ocd_id):
+        # url = "%sboundaries/%s/simple_shape" % (self.base_url, boundary_id)
+        url = "{}{}?apikey={}".format(
+            settings.BOUNDARY_SERVICE_URL,
+            ocd_id,
+            settings.API_KEY,
+        )
+        data = json.load(urllib2.urlopen(url))
+        geometries = filter(
+            lambda x: x['boundary_set']['name'] in ACTIVE_BOUNDARY_SETS,
```

Comment:
the server should be limiting based on time, rather then `ACTIVE_BOUNDARY_SETS`. I'd love to cut that.